docs/organizer-guide/meetups/starting.rst

```diff
@@ -14,176 +13,186 @@ It contains a lot of the knowledge that we have put here:
 
-When you're done, `contact us <mailto:conf@writethedocs.org>`_ or on `Slack <https://writethedocs.slack.com/>`_: we will add your group to the :doc:`/meetups/index` list on our website.
+Find a Place to Meet
+--------------------
 
-Find a Place to Meet (Sponsors)
--------------------------------
+Ideally, you can start by setting up a Meetup at your corporate facility. If your facilities aren't convenient to your potential community of documentarians, find "documentarians" or "community managers" for centrally located companies in your area. You can typically find these people at other technical meetups/conferences in your area.
 
-Start by asking companies you're close to: it could be your company, your friends' companies, etc.
-
-If it's not working, contact other tech companies in your town. Job boards, tech conferences websites, and other tech meetups are also a good way to start. If you're going to tech conferences, go to sponsor booths and ask them if they would be interested in sponsoring your meetup.
-
-It's easier to know someone inside the company you want to contact. If you don't know anyone, you can use this email template:
+Alternatively, the electronic version of "cold-calling" can work. We provide this email template for your reference:
 
 .. code-block:: jinja
 
     Hello,
 
-    My name is {{ name }} and I am the organizer of Write the Docs
-    {{ city name }} meetup {{ link to meetup page }}. Write the Docs
-    {{ writethedocs.org }} is a global community of technical writers,
-    designers, typesetters, developers, support teams, marketers and
-    many others interested in documentation.
+    I am the organizer of Write the Docs {{ city name }} meetup {{ link to meetup page }}. Write the Docs
+    {{ writethedocs.org }} is a global community of people interested in excellent documentation.
 
-    We're looking for a sponsor to host our {{ first/next }} meetup.
-    This could be a great opportunity to reach new writers and to
-    develop your team skills.
+    We'd like your help! If you're interested in improving your documentation, help us. We'll provide a resource with different experiences, and potentially
+    writers who can help with your documentation.
 
-    If you have any questions, please do not hesitate to contact me.
+    If you have any questions, please do not hesitate to contact us.
 
     Regards,
     {{ name }}
 
-Can't Find a Place to Meet?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Alternative Meetup Locations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you can't find a company to host your meetup, try something different: `meet in a cafe <http://www.meetup.com/Write-The-Docs-Seattle/events/231890982/>`_ to work on a common project, organize a picnic or do an "after-work" in a bar. Networking is an important part of meetups and it's better to meet every month than to skip one because you didn't find a sponsor.
```

Comment:
We should try and keep links to some of the existing meetups that other folks have done, to give folks inspiration.